lib/incito-browser/incito.js

```diff
@@ -48,6 +48,14 @@ export default class Incito extends MicroEvent {
             this.el.style.color = theme.text_color;
         }
 
+        if (isDefinedStr(theme.style)) {
+            this.styleEl = document.createElement('style');
+
+            this.styleEl.innerText = theme.style;
+
+            document.head.appendChild(this.styleEl);
+        }
```

Comment:
Incito team needs to be super diligent not to mess with ETA & client styles, of course

Reply:
@klarstrup Indeed. I wish we could scope the CSS. There was a proposal for `scoped` style attribute but it's deprecated. Do you have a better approach? This seemed like the best one to me.

Reply:
We could scope the CSS like this: 

1. put the incito ID as a class on the incito mount point i.e. `<div class="sgn__incito incito__moBP88ob"></div>`
2. have bumblebee prefix all the styles with `.incito__moBP88ob <actual selector goes here>`(i.e. `.incito__moBP88ob h1 {font-size: xxx-large;})`
    3. or modify `theme.style` here to add `.incito__moBP88ob` to all rules (will require a lot of parsing and string munging)

Reply:
That could work but if we have a property for global styles and we want to prepend an Incito ID in front, wouldn't we have to parse all the CSS and do string munging, as you say?